src/modules/organization/event/components/table/Table.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="flex justify-between my-2">
+    <div class="flex space-x-3">
+      <BaseSearch class="h-4" :placeholder="'Search'" @search="handleSearch" />
+      <BaseButton
+        class="flex justify-center items-center // h-4 px-2 space-x-1 flex-shrink-0"
+        @click="approveAllRequest"
+      >
+        <div>Approved all</div>
+        <base-icon class="w-3 h-3">
+          <check-circle-icon />
+        </base-icon>
+      </BaseButton>
+    </div>
+
+    <SingleNameSelect
+      :optionNames="sortBy"
+      :optionValues="sortByVal"
+      v-model="sortOption"
+    />
+  </div>
+  <div class="p-3 bg-white // rounded-2xl // w-full">
+    <table class="table-auto // w-full">
+      <thead>
+        <tr>
+          <th
+            v-for="headerKey in headerKeys"
+            :key="headerKey"
+            class="pb-1 // border-b-2 border-blue-11 // text-left"
+          >
+            {{ header[headerKey] }}
+          </th>
+          <th class="pb-1 // border-b-2 border-blue-11 // text-right">
+            Action
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(data, index) in filteredData" :key="index">
+          <td
+            v-for="(header, index) in headerKeys"
```

Comment:
what is header, Non and New don't understand what it is. Can we have a better name for it?